src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -166,29 +189,33 @@ public void setLog10PError(double log10PError) {
     // Working with attributes
     //
     // ---------------------------------------------------------------------------------------------------------
+    /** removes all attributes */
     public void clearAttributes() {
         attributes = new HashMap<String, Object>();
     }
 
     /**
+     * return an unmodifiable Map of attributes
      * @return the attribute map
      */
     public Map<String, Object> getAttributes() {
         return Collections.unmodifiableMap(attributes);
     }
 
     // todo -- define common attributes as enum
-
-    public void setAttributes(Map<String, ?> map) {
+    /** clear all attributes, and set 'map' as attributes */
+    public void setAttributes(final Map<String, ?> map) {
         clearAttributes();
         putAttributes(map);
     }
 
-    public void putAttribute(String key, Object value) {
+    /** shortcut of <code>putAttribute(key, value, false)</code> */
```

Comment:
Add proper javadoc for this method (and for the other methods where you've written "shortcut of x")